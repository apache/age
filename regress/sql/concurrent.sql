\! python3 regress/python/test_concurrent.py
