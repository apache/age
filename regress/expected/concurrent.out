\! python3 regress/python/test_concurrent.py
Result: OK

Result: OK

Result: OK

Result: OK

Result: OK

All threads have finished execution.
